docker/env/hydra.sh

```diff
@@ -248,6 +248,7 @@ function run_in_docker () {
         -v /tmp:/tmp \
         -v /var/tmp:/var/tmp \
         -v "${HOME_DIR}:${HOME_DIR}" \
+        --tmpfs "${HOME_DIR}/.local:exec,uid=$(id -u ${USER}),gid=$(id -g ${USER})" \
```

Comment:
This code also gets run on builders.
This is the reason I added TEST_ID to the source dir name.

Reply:
Also, running locally 2 different hydra commands for different test runs now will overlap.

Reply:
what is gonna overlap with this, --tmpfs mean it would be generate on every docker run command inside the docker.

it won't be depended on anything external to that specific run.

Reply:
It can be simplified by using the existing var:

```
USER_ID=$(id -u "${USER}"):$(id -g "${USER}")
```

Reply:
I thought of that, but it's not the exact same syntax
one is `1000:1000` and one `uid=1000,gid=1000`

so it's better left as is.